src/core/job.c

```diff
@@ -801,15 +801,15 @@ _pure_ static const char *job_get_done_status_message_format(Unit *u, JobType t,
 static const struct {
         const char *color, *word;
 } job_print_done_status_messages[_JOB_RESULT_MAX] = {
-        [JOB_DONE]        = { ANSI_OK_COLOR,         "  OK  " },
-        [JOB_TIMEOUT]     = { ANSI_HIGHLIGHT_RED,    " TIME " },
-        [JOB_FAILED]      = { ANSI_HIGHLIGHT_RED,    "FAILED" },
-        [JOB_DEPENDENCY]  = { ANSI_HIGHLIGHT_YELLOW, "DEPEND" },
-        [JOB_SKIPPED]     = { ANSI_HIGHLIGHT,        " INFO " },
-        [JOB_ASSERT]      = { ANSI_HIGHLIGHT_YELLOW, "ASSERT" },
-        [JOB_UNSUPPORTED] = { ANSI_HIGHLIGHT_YELLOW, "UNSUPP" },
+        [JOB_DONE]        = { ANSI_OK_COLOR,                     "  OK  " },
+        [JOB_TIMEOUT]     = { ANSI_HIGHLIGHT_RED,                " TIME " },
+        [JOB_FAILED]      = { ANSI_HIGHLIGHT_RED,                "FAILED" },
+        [JOB_DEPENDENCY]  = { ANSI_HIGHLIGHT_YELLOW_REPLACEMENT, "DEPEND" },
+        [JOB_SKIPPED]     = { ANSI_HIGHLIGHT,                    " INFO " },
+        [JOB_ASSERT]      = { ANSI_HIGHLIGHT_YELLOW,             "ASSERT" },
+        [JOB_UNSUPPORTED] = { ANSI_HIGHLIGHT_YELLOW,             "UNSUPP" },
```

Comment:
forgot some here?

Reply:
there are some other uses in our codebase of yellow too, for example in systemd-analyze

Reply:
Oops. Fixed this here. systemd-analyze was already updated.